sdk/provisioning/Azure.Provisioning.AppConfiguration/src/Azure.Provisioning.AppConfiguration.csproj

```
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Contains the core functionality for defining Azure infrastructure with dotnet code.</Description>
```

We should use whatever tag at the top of the README here.
```suggestion
    <Description>Azure.Provisioning.AppConfiguration simplifies declarative resource provisioning in .NET for Azure App Configuration.</Description>
```